app/models/program.rb

```diff
@@ -49,7 +51,7 @@ def confirmed
 #   validates :conference_id, presence: true, uniqueness: true
   validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 10 }
 
-  before_create :create_event_types
+  after_create :create_event_types
```

Comment:
I wasn't tracking DifficultyLevel.I tried it now locally and it has the same issue.So if were to similarly track DifficultyLevel similar change should be made for [https://github.com/openSUSE/osem/blob/master/app/models/program.rb#L53]( https://github.com/openSUSE/osem/blob/master/app/models/program.rb#L53)